layers/+tools/shell/README.org

```diff
@@ -233,3 +279,13 @@ in the current buffer instead of a popup.
 | Key Binding        | Description                                |
 |--------------------+--------------------------------------------|
 | ~SPC m H~ or ~M-l~ | shell commands history using a helm buffer |
+
+
+
+
+** Vterm (EXPERIMENTAL)
```

Comment:
Reduce the four empty lines above to one.
Github only shows them as one.
https://github.com/syl20bnr/spacemacs/blob/f4a8ac0f7c9244b90a3a64a5a380eee0442caf96/layers/%2Btools/shell/README.org